src/any_agent/frameworks/any_agent.py

```diff
@@ -122,10 +126,15 @@ async def create_async(
         agent_config: AgentConfig,
         managed_agents: list[AgentConfig] | None = None,
         tracing: TracingConfig | None = None,
+
     ) -> AnyAgent:
         """Create an agent using the given framework and config."""
         agent_cls = cls._get_agent_type_by_framework(agent_framework)
-        agent = agent_cls(agent_config, managed_agents=managed_agents, tracing=tracing)
+        agent = agent_cls(
```

Comment:
I suspect this is being changed because a disparity in the autoformater being used (maybe in the IDE). If it was not failing in `pre-commit ruff-format` before, it would be better to not introduce formatting changes

Reply:
I was unsure if I should add the additional exporters in the tracing config (being references and not serializable data items) so the format changed as a result. @njbrake @daavoo I'd like to get your thoughts on that: should runtime object references be present in a pydantic config object? Or handled as separate params?